packages/react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -107,6 +107,7 @@ export class TooltipHostBase extends React.Component<ITooltipHostProps, ITooltip
             directionalHintForRTL={directionalHintForRTL}
             calloutProps={assign({}, calloutProps, {
               onDismiss: this._hideTooltip,
+              onFocus: this._onTooltipContentFocus,
```

Comment:
Do you also need `onBlur` to dismiss the tooltip when it loses focus (e.g. tabbing out of the tooltip after clicking the link)? Or is that handled by the callout already?

Reply:
Yeah, that's handled by `Callout` already.